app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    text = command.split(" ")
```

Comment:
According to the checklist, you should ensure that the input string contains exactly three elements separated by spaces and that the first element is 'cp'. Consider adding a check for this condition before proceeding with the rest of the function logic.